lib/matplotlib/colorbar.py

```diff
@@ -180,6 +180,9 @@
 to which the colorbar is attached; but it is a manual method requiring
```

Comment:
I'd actually drop this part.  `*shtrink*` means to shrink for many reasons.  I routinely shrink my colorbars for esthetic reasons that have nothing to do w/ the size of the mappable.